<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductInDialog</class>
 <widget class="QDialog" name="ProductInDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font: bold 15px;
}
QDialog
{
background-image:url(&quot;:/Theme/BackImage/DialogBack_1.jpg&quot;)}</string>
  </property>
  <widget class="QComboBox" name="ProductChooseBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择商品：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前存量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CountHavenEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ImageLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>20</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>进货数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CountEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>进货单价：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InPriceEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>279</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>预计出售单价：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutPriceEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>339</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>确认进货信息并提交</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
